services/user_service/database_connector.py

```diff
@@ -21,31 +22,29 @@ def get_all_users(self):
             return self.__users_ref.get()
         except FirebaseError as e:
             print("###Connection lost. EROR received : " + str(e))
-            return list()
-    
+        return list()
 
     def get_user_by_id(self, id):
         try:
             if self.__users_ref.child(id).get() is None:
-                return list()
+                return None
```

Comment:
Can be removed

Reply:
agree. when error will happen it will return None a few lines below

Reply:
agree. when error will happen it will return None a few lines below